Sources/SocksCore/Socket.swift

```diff
@@ -32,6 +32,11 @@ extension RawSocket {
             throw SocksError(.closeSocketFailed)
         }
     }
+
+    @available(*, deprecated: 1.2.6, renamed: "isClosed", message: "use isClosed instead")
```

Comment:
This is a breaking change either way since protocol conformers would have to update, I'd opt for removing the deprecation warning and letting the code choke.

This PR might hang for a bit until we're ready to combine it w/ more 2.0 stuff, but should be soon